master/internal/provisioner/aws.go

```diff
@@ -192,7 +188,6 @@ func (c *awsCluster) launch(
 	if instanceNum <= 0 {
 		return
 	}
-	ctx.Log().Infof("launching %d EC2 instances", instanceNum)
```

Comment:
Why remove this? It seems like a useful log

Reply:
I have also added logs in the `provisioner.provision()`